app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,19 @@
+class RelationshipsController < ApplicationController
+  before_action :require_user
+
+  def index
+    @relationships = current_user.following_relationships
+  end
+
+  def create
+    leader = User.find(params[:leader_id])
+    Relationship.create(leader_id: params[:leader_id], follower: current_user) if current_user.can_follow?(leader)
```

Comment:
Oh I didn't notice that. Thanks! 

Tim Beiko 

> On Aug 23, 2015, at 22:11, Brandon Conway <notifications@github.com> wrote:
> 
> In app/controllers/relationships_controller.rb:
> 
> > @@ -0,0 +1,19 @@
> > +class RelationshipsController < ApplicationController
> > +  before_action :require_user
> > +
> > +  def index
> > +    @relationships = current_user.following_relationships
> > +  end
> > +
> > +  def create
> > +    leader = User.find(params[:leader_id])
> > +    Relationship.create(leader_id: params[:leader_id], follower: current_user) if current_user.can_follow?(leader)
> You can also do:
> 
> Relationship.create(leader: leader, follower: current_user) if current_user.can_follow?(leader)
> since you have the leader object.
> 
> —
> Reply to this email directly or view it on GitHub.
>